packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/ScopeHandlerFactoryImpl.ts

```diff
@@ -45,10 +45,11 @@ import type { CustomScopeType, ScopeHandler } from "./scopeHandler.types";
  */
 export class ScopeHandlerFactoryImpl implements ScopeHandlerFactory {
   constructor(private languageDefinitions: LanguageDefinitions) {
+    this.tryCreate = this.tryCreate.bind(this);
     this.create = this.create.bind(this);
   }
 
-  create(
+  tryCreate(
```

Comment:
I believe we usually use `maybe` for things like this, eg `maybeCreate`

Reply:
Fixed